package_linter.py

```diff
@@ -2433,11 +2421,10 @@ def admin_has_to_finish_install(self):
         if os.system(cmd) == 0:
             yield Warning("It looks like this app requires the admin to finish the install by entering DB credentials. Unless it's absolutely not easily automatizable, this should be handled automatically by the app install script using curl calls, or (CLI tools provided by the upstream maybe ?).")
 
-
     @test(only=["install", "upgrade"])
     def deprecated_replace_string(self):
         cmd1 = "grep -Ec 'ynh_replace_string' '%s' || true" % self.path
-        cmd2 = "grep -Ec 'ynh_replace_string.*__\w+__' '%s' || true" % self.path
+        cmd2 = "grep -Ec 'ynh_replace_string.*__\\w+__' '%s' || true" % self.path
```

Comment:
yeah i'm not sure what the heck, but apparently in Python, both `s = "\w"` and `s = "\\w"` will result in the same string ... probably because it doesn't make sense to escape a `w`, so Python understands that it's a litteral `\` .. interesting ;D